src/09-word-problems.js

```diff
@@ -127,7 +162,26 @@ function compareLocations(address1, address2) {}
     //>   },
     //> ];
  */
-function gradeAssignments(assignments) {}
+function gradeAssignments(assignments) {
+  for(let key in assignments){
+    console.log(assignments[key].score.received === assignments[key].score.max);
+    if(assignments[key].kind === "PASS-FAIL" && assignments[key].score.received === assignments[key].score.max){
```

Comment:
We also may consider breaking these chained statements on to new lines after the`&&`s we use